web/admin/application/library/IvozProvider/Klear/Ghost/NotificationTemplate.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Ivoz\Core\Application\Service\DataGateway;
+use Ivoz\Provider\Domain\Model\Brand\Brand;
+use Ivoz\Provider\Domain\Model\Brand\BrandDto;
+use Ivoz\Provider\Domain\Model\Company\CompanyDto;
+use IvozProvider\Utils\SizeFormatter;
+
+class IvozProvider_Klear_Ghost_NotificationTemplate extends KlearMatrix_Model_Field_Ghost_Abstract
```

Comment:
Is this finished?

Reply:
Yes. That's the intented code. Because Klear.